src/DocumentFormat.OpenXml/Properties/launchSettings.json

```diff
@@ -0,0 +1,7 @@
+{
+  "profiles": {
+    "DocumentFormat.OpenXml": {
```

Comment:
This file and the other one appear unrelated to the change. I believe this is for vs code. If so, happy to take it but would like the rest of the projects supported as well. Best in a separate pr though